backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/spreadsheet/process/ExportToSpreadsheetProcess.java

```diff
@@ -56,6 +58,9 @@ protected String doIt()
 		final String sql = getSql();
 		final Evaluatee evalCtx = getEvalContext();
 
+		final int recordId = getRecord_ID();
```

Comment:
you don't have to load this so early. you can do it when u are updating it (below)